commonui/build.gradle

```diff
@@ -11,6 +11,7 @@ apply from: "$rootProject.projectDir/android-library.gradle"
 apply from: "$rootProject.projectDir/publish.gradle"
 
 dependencies {
+    api "com.hyperwallet.android:insight-sdk:$hyperwalletInsightVersion"
```

Comment:
I'd rather have the dependency added directly to each ui module (for now just to the transfermethodui) instead of having it indirectly being used by those modules through the common module. Then we would use implementation instead of api...
@peter-joseph what do you think?